src/Features/CSharp/Portable/UseLocalFunction/CSharpUseLocalFunctionCodeFixProvider.cs

```diff
@@ -46,18 +46,18 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
         {
             var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
 
-            var localDeclarationToLambda = new Dictionary<LocalDeclarationStatementSyntax, LambdaExpressionSyntax>();
+            var localDeclarationToFunction = new Dictionary<LocalDeclarationStatementSyntax, AnonymousFunctionExpressionSyntax>();
             var nodesToTrack = new HashSet<SyntaxNode>();
             var explicitInvokeCalls = new List<MemberAccessExpressionSyntax>();
             foreach (var diagnostic in diagnostics)
             {
                 var localDeclaration = (LocalDeclarationStatementSyntax)diagnostic.AdditionalLocations[0].FindNode(cancellationToken);
-                var lambda = (LambdaExpressionSyntax)diagnostic.AdditionalLocations[1].FindNode(cancellationToken);
+                var function = (AnonymousFunctionExpressionSyntax)diagnostic.AdditionalLocations[1].FindNode(cancellationToken);
 
-                localDeclarationToLambda[localDeclaration] = lambda;
+                localDeclarationToFunction[localDeclaration] = function;
```

Comment:
still not sure about the name "function"... but "localDeclarationToAnonymousFunction" is just too long

Reply:
'localDeclarationToAnonymousFunction' seems fine to me.